Pipfile

```diff
@@ -0,0 +1,19 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+six = "*"
```

Comment:
As these are duplicated from setup.py, maybe it would make sense to remove them and install from setup.py:

```shell
$ pipenv install -e .
```

Although, I'm not too sure this is right.